config/runtime.exs

```diff
@@ -26,7 +33,8 @@ auth_token =
         "Xyf..."
     """
 
-store = {MiniRepo.Store.Local, root: {:mini_repo, "data"}}
+root = System.get_env("MINI_REPO_STORE_ROOT") || {:mini_repo, "data"}
```

Comment:
as reported in https://github.com/wojtekmach/mini_repo/issues/24#issue-564154941, the `MINI_REPO_STORE_ROOT` isn't (and shouldn't!) be used in the endpoint https://github.com/wojtekmach/mini_repo/blob/1b7a6bcbcc80f69ebdda36697c8d252a6cba9d9d/lib/mini_repo/endpoint.ex#L10.